### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards> FlightItinerary.json]: A hidden text label is not provided with 'Source' and 'Arrival' destination information with flights.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7830]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Browser: Edge Dev (106.0.1356.0)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2206.2001
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ windows+ enter key.
3. Navigate to the 'FlightItinerary.json' through tab key and activate it.
4. Now, navigate to 'San Francisco' and 'Amsterdam through up/down arrow key.
5. Observe the issue.

**Actual Result:**
For 'Source' and 'Arrival' destination, a hidden text label is not defined. Screen reader is only announcing as 'San Francisco' and 'Amsterdam'.

**Expected Result:**
For 'Source' and 'Arrival' destination, a hidden text label should be defined. Screen reader should announce as 'Source San Francisco' and 'Arrival destination Amsterdam' OR Visual label as 'Source' and 'Arrival destination' should be defined above the particular flights.

**User Impact:**
Screen reader user will get impacted if hidden text label is not provided due to which they will not be able to understand which flight information is for source or arrival and will have difficulty in making track of the content properly.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/03.03.02

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

Screenshots

### Screenshots

https://user-images.githubusercontent.com/104483005/189759254-8687d4f0-65a6-4260-b3d7-d5db1f90de1f.mp4

### Card JSON

```JSON
NA
```

**Comment by vaishali1397** [comment]
Issue is repro'ing in below test environment, hence reactivating the bug.
Test Environment:
OS version: 22H2(Build 25272.1000)
Application: Adaptive Cards Visualizer (3.0.2210.26001 (latest) build.)

**Comment by JeanRoca** [comment]
This issue has been fixed and will be included in our next release. I am closing this issue for now. Once new version is available, if the issue persists please open a new issue to track.

**Comment by vaishali1397** [comment]
Issue has been fixed in below test Environment i.e hidden text as '**Departs From'** and **Arrives at**' is provided now, hence Adding A11yTTValidated tag.
**Test Environment:**
OS Version: 22H2 (Build 25314. 1010)
Application: Adaptive Card Visualizer (Version 3.2.2303.22003)

**
https://user-images.githubusercontent.com/104483005/228219419-23c337bd-16e6-43a7-8ff2-180895bed1b9.mp4
** Adaptive cards UWP~ Fixed issue.mp4
